### ISSUE TITLE
[PW-3329] Call to undefined method ModuleFrontController::ajaxRender

### ISSUE THREAD

**Issue by mehdi-ghezal** [Adyen/adyen-prestashop#132]
**Describe the bug**
After installing the module and following the guideline about configuration for the test environment, when I pay with a test card, the AJAX request sent by the module lead to a PHP  Fatal Error.

**To Reproduce**

1. Create a new account on Adyen
1. Install the Prestashop module directly from Prestashop Backoffice
1. Follow the setup instruction: https://docs.adyen.com/plugins/prestashop
1. Proceed to checkout and "Pay by Carte bancaire"
1. Use the test card from: https://docs.adyen.com/development-resources/test-cards/test-card-numbers ; I use this one: 4035 5010 0000 0008 - Visa Debit / Cartes Bancaires - FR - 03/2030 - 737

**Expected behavior**
Payement successful or at least a feedback error ; obviously not a Fatal Error.

**Logs**

```
PHP Fatal error:  Uncaught Error: Call to undefined method ModuleFrontController::ajaxRender() in ...../modules/adyenofficial/controllers/FrontController.php:155
Stack trace:
    #0 ...../modules/adyenofficial/controllers/front/Payment.php(191): Adyen\PrestaShop\controllers\FrontController->ajaxRender('{"action":"erro...')
    #1 ...../classes/controller/Controller.php(242): AdyenOfficialPaymentModuleFrontController->postProcess()
    #2 ...../classes/Dispatcher.php(428): ControllerCore->run()
    #3 ...../index.php(28): DispatcherCore->dispatch()
    #4 {main}
        thrown in ...../modules/adyenofficial/controllers/FrontController.php on line 155
``` 

**Desktop (please complete the following information):**
Not applicable/relevant

**Smartphone (please complete the following information):**
Not applicable/relevant

**Additional context**
Linux Debian 9.13 with PHP 7.0 (PHP FPM)
Prestashop 1.7.3

**Comment by cyattilakiss** [comment]
Hi @mehdi-ghezal ,

Thank you for raising this issue.
I could verify that it's indeed an issue on the PrestaShop version 1.7.3 and was fixed in the version 1.7.5.
I will create and assign an internal ticket to fix this incompatibility issue in those versions asap.

Best,
Attila
Adyen

**Comment by cyattilakiss** [comment]
Hi @mehdi-ghezal ,

The issue has been fixed in the release 3.2.0
I will close this issue for now but please let us know if you still face this issue or have any further questions.

Best,
Attila
Adyen
